### ISSUE TITLE
Unable to pip install mediapipe==0.9.0

### ISSUE THREAD

**Issue by WissemBT** [google/mediapipe#5396]
### OS Platform and Distribution

Linux Ubuntu 20.04; Linux Ubuntu 22.04

### Programming Language and version

Python 3.8.10 ; Python 3.10.12

### Installed using virtualenv? pip? Conda?(if python)

pip install mediapipe==0.9.0

### MediaPipe version

0.9.0

### Android AAR (if android)

None

### Describe the problem

Installation of mediapipe==0.9.0 fails when pip install mediapipe==0.9.0 as version is no longer available.

### Complete Logs

```shell
We are currently unable to install mediapipe==0.9.0, which was previously available and used in both our local development environments and CI pipeline. Attempts to install this specific version now fail with error messages indicating that the version is no longer available in the repository : 
ERROR: Could not find a version that satisfies the requirement mediapipe==0.9.0 (from versions: 0.9.1.0, 0.9.2.1, 0.9.3.0, 0.10.0, 0.10.1, 0.10.2, 0.10.3, 0.10.5, 0.10.7, 0.10.8, 0.10.9, 0.10.10, 0.10.11, 0.10.13, 0.10.14)
ERROR: No matching distribution found for mediapipe==0.9.0
The Last successful installation was in 7 may 2024.
```

**Comment by kuaashish** [comment]
Hi @WissemBT,

We had to remove this version from our PyPi package because we are consistently hitting our storage limit. As a result, you can not install it from PyPi anymore. If you have the files saved, you can install it manually. Unfortunately, we can not provide the wheel files since we do not have them elsewhere. We are sorry for any inconvenience and won't be able to help further with this issue

Thank you!!

**Comment by WissemBT** [comment]
Hi @kuaashish ,

Thank you for your response and clarification, I understand the constraints regarding storage limits on PyPI.
Before closing the issue, could you please advise on the recommended approach for migrating to a newer version of mediapipe? Are there specific versions that would have minimal migration issues or compatibility concerns from version 0.9.0?

**Comment by kuaashish** [comment]
Hi @WissemBT,

The most recent version we have released is 0.10.14, which includes support for Python 3.12 while discontinuing support for Python version 3.8. The supported Python [versions](https://developers.google.com/mediapipe/solutions/setup_python#supported_platforms_and_versions) now range from 3.9 to 3.12. You can easily install this version using the command:

`pip install mediapipe`

Alternatively, if the installation using the above command fails, you can try manually install it by running:

`pip install mediapipe==0.10.14`

We always recommend using the latest available version, which is 0.10.14. If you encounter any issues, please report them, and we will be happy to assist you further.
`
Thank you!!

**Comment by WissemBT** [comment]
Hi @kuaashish ,
Thank you for your answer, I am going to close the issue.
